Editor/Window/Components/ProfileSelector.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using AmazonGameLift.Editor;
+using Editor.Resources.EditorWindow;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace GameLiftPlugin.Editor
+{
+    public class ProfileSelector : VisualElement
+    {
+        public new class UxmlFactory : UxmlFactory<ProfileSelector> { }
+
+        private DropdownField _dropdown => this.Q < DropdownField>("Dropdown");
+        
+        public ProfileSelector()
+        {
+            var asset = Resources.Load<VisualTreeAsset>("EditorWindow/Components/ProfileSelector");
+            asset.CloneTree(this);
+            
+            LocalizeText();
+
+            _dropdown.choices = new List<string>(); // TODO: Load choices from plugin state
```

Comment:
What's blocking us implementing this right now? Is there a follow-up PR for this item?

Reply:
Sorted